pkg/kubelet/status/manager.go

```diff
@@ -354,49 +354,56 @@ func (m *manager) syncPod(uid types.UID, status versionedPodStatus) {
 		// If the Pod is deleted the status will be cleared in
 		// RemoveOrphanedStatuses, so we just ignore the update here.
 		return
+	} else if err != nil {
+		// We failed to update status, wait for periodic sync to retry.
+		glog.Warningf("Failed to update status for pod %q: %v", format.Pod(pod), err)
```

Comment:
This might lead to false positives in the case of deleted pods, like the comment in line 354 states. Should we lower the logging level or at-least include the fact that the pod might be deleted?